From 83f416fa91fd62fe14b6bed0c3072b5fc53822eb Mon Sep 17 00:00:00 2001
From: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
Date: Thu, 15 Jul 2021 11:24:30 +1000
Subject: [PATCH v4 1/2] include: Updated _IMAGEHLP_MODULEW64 stucture

This aligns the structure to the be the same found in the 10.0.19041.0 SDK.

Signed-off-by: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
---
 include/dbghelp.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/dbghelp.h b/include/dbghelp.h
index 1b234899195..303c0109d1f 100644
--- a/include/dbghelp.h
+++ b/include/dbghelp.h
@@ -280,6 +280,8 @@ typedef struct _IMAGEHLP_MODULE64
     BOOL                        TypeInfo;
     BOOL                        SourceIndexed;
     BOOL                        Publics;
+    DWORD                       MachineType;
+    DWORD                       Reserved;
 } IMAGEHLP_MODULE64, *PIMAGEHLP_MODULE64;
 
 typedef struct _IMAGEHLP_MODULEW64
@@ -307,6 +309,8 @@ typedef struct _IMAGEHLP_MODULEW64
     BOOL                        TypeInfo;
     BOOL                        SourceIndexed;
     BOOL                        Publics;
+    DWORD                       MachineType;
+    DWORD                       Reserved;
 } IMAGEHLP_MODULEW64, *PIMAGEHLP_MODULEW64;
 
 #if !defined(_IMAGEHLP_SOURCE_) && defined(_IMAGEHLP64)
-- 
2.30.2

